mypy/ordered_dict.py

```diff
@@ -0,0 +1,9 @@
+# OrderedDict is kind of slow, so for most of our uses in Python 3.6
+# and later we'd rather just use dict
+
+import sys
+
+if sys.version_info <= (3, 5):
```

Comment:
This won't work right because in reality version_info is something like `(3, 6, 10, "final", 0)`, which for any kind of 3.5 is `> (3, 5)`. You need to write `< (3, 6)` instead.

Reply:
Yeah so it seems. Fun fact: I think mypy doesn't understand this.

Reply:
We put checks against this sort of thing in flake8-pyi (https://github.com/ambv/flake8-pyi/blob/master/pyi.py#L230) but obviously that doesn't help here since this isn't a stub. Maybe mypy itself should have the same checks.